addons/laser/stringtable.xml

```diff
@@ -63,5 +63,11 @@
             <Chinesesimp>雷射 - 循环切换雷射码 下</Chinesesimp>
             <Chinese>雷射 - 循環切換雷射碼 下</Chinese>
         </Key>
+        <Key ID="STR_ACE_Laser_LSTOn">
+            <English>Laser Spot Tracker: On</English>
```

Comment:
I'm sure there is already a vanilla translation for On/Off that could be reused here, and get rid of the mostly duplicate entries here.

Reply:
Where would I find it?

Reply:
either unpack and search the vanilla stringtables dta/languagecore.pbo or 
https://synixebrett.github.io/stringtables/
STR_ACTION_LASER_OFF
is a allcaps OFF
STR_3DEN_Attributes_Radar_RadarOff_text
is a normal Off

Meh... Can't find a "On" though

Reply:
https://github.com/acemod/ACE3/blob/7cc168e1b953c13dd52bda2db486aa3d2381f2e1/addons/map/stringtable.xml#L388
Here are On and Off, but I see the german translation for example wouldn't fit here as it doesn't mean "enabled/on" but more like "one" the number.

Reply:
https://github.com/acemod/ACE3/blob/d8f18c685b89e07730be1b9bff70dc6010ce2ee5/addons/viewdistance/stringtable.xml#L255
Same...
https://github.com/acemod/ACE3/blob/35e4c2ae359ef54038423f0ace67306771c63364/addons/microdagr/stringtable.xml#L118
same... did we just copy the same On/Off translations into multiple components? 🤔